start_cerberus.py

```diff
@@ -133,6 +134,24 @@ def main(cfg):
                              "assuming that the nodes are ready")
                 watch_nodes_status = True
 
+            # Monitor cluster operators status
+            if watch_operators:
+                operators_status = runcommand.invoke("oc get co -o yaml")
```

Comment:
Maybe use kubectl instead of oc? This way it works both on kubernetes and OpenShift clusters. Thoughts?

Reply:
@mffiedler what are your thoughts on this. I was following the function in the issue. 

I thought cluster operators were specific to openshift;  is it available in kubernetes as well?

Reply:
We should use kubectl where possible to keep the tool compatible with upstream.   @chaitanyaenr Cerberus and Kraken are going to evolve into areas that are necessarily OpenShift specific.   Should we have a top level flag that can be used to determine if a  Cerberus monitor or Kraken scenario is only appropriate if running in OpenShift?

Reply:
Agreed. We can explicitly mention in the docs/config about the things which are specific to OpenShift for users to avoid using them when running the tool on Kube. This also ties to the following issue: https://github.com/openshift-scale/cerberus/issues/44.

We should make use of functions available in kubernetes client ( https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/AppsV1Api.md ) as much as possible like how we are doing today ( https://github.com/openshift-scale/cerberus/blob/master/cerberus/kubernetes/client.py ). We can look into using OpenShift client as need arises I guess. 

Thoughts?